res/database/Default/database_news.xml

```diff
@@ -2,6 +2,2994 @@
+			<title>
+				<de>Neue Computerära begonnen</de>
+				<en>New computer era begun</en>
+				<pl>Rozpoczęła się nowa era komputerów</pl>
+			</title>
+			<description>
+				<de>Der amerikanische Computerproduzent Cormorano stellt seinen Heimrechner „Eterna“ vor. Seine Grafikleistung ist einfach atemberaubend.</de>
+				<en>The American computer manufacturer Cormoran presents its home computer "Eterna". Its graphics performance is simply breathtaking.</en>
+				<pl>Amerykański producent komputerów Cormoran prezentuje swój komputer domowy "Eterna". Jego wydajność graficzna po prostu zapiera dech w piersiach.</pl>
+			</description>
+			<data genre="3" price="1.0" quality="45" fictional="0" flags="2" happen_time="4,1985,7,23" />
+		</news>
+
+			<!-- 1985-08 -->
+
+		<news id="fdb578af-f9d0-4c33-8c29-5a75605fae0e" type="0" creator="8936" created_by="jorgaeff">
+			<title>
+				<de>2000-Jahrfeier in Augsburg</de>
+				<en>2000th anniversary celebration in Augsburg</en>
+				<pl>Obchody 2000-lecia w Augsburgu</pl>
+			</title>
+			<description>
+				<de>Die Stadt Augsburg in Bayern feiert ihren 2000. Geburtstag. Sie ist eine der ältesten Städte Deutschlands und war im 16. Jahrhundert Heimat der wohlhabenden Kaufmannsfamilie Fugger.</de>
+				<en>The city of Augsburg in Bavaria celebrates its 2000th birthday. It is one of the oldest cities in Germany and was home to the wealthy merchant family Fugger in the 16th century.</en>
+				<pl>Miasto Augsburg w Bawarii obchodzi swoje 2000 urodziny. Jest to jedno z najstarszych miast w Niemczech i było domem dla bogatej rodziny kupieckiej Fugger w XVI wieku.</pl>
+			</description>
+			<data genre="5" price="0.25" quality="40" fictional="0" flags="2" happen_time="4,1985,8,8" /> <!-- using date of the annual 'Friedensfest' -->
+		</news>
+
+			<!-- 1985-09 -->
+
+		<news id="52a80873-d74e-4d41-9aca-ac16087f4b14" type="0" thread_id="cfb216de-72a7-431e-b8cf-8c8bbfc4a6a8" creator="7430" created_by="Sjaele" comment="https://en.wikipedia.org/wiki/Wreck_of_the_Titanic#Discovery">
+			<title>
+				<de>Titanic-Wrack entdeckt</de>
+				<en>Titanic wreck discovered</en>
+				<pl>Odkryto wrak Titanica</pl>
+			</title>
+			<description>
+				<de>Forscher entdeckten auf einer speziellen Expedition das Wrack in 3800 Metern Tiefe. 21 km ostsüdöstlich von der im Notruf angebenen Position.</de>
+				<en>On a special expedition, researchers discovered the wreck at a depth of 3800 metres. 21 km east-southeast of the location indicated in the distress call.</en>
+				<pl>Podczas specjalnej ekspedycji badacze odkryli wrak na głębokości 3800 metrów. 21 km na wschód i południowy wschód od miejsca wskazanego w wezwaniu pomocy.</pl>
+			</description>
+			<data genre="3" price="1.0" quality="52" fictional="0" flags="2" happen_time="4,1985,9,1" />
+			<effects>
+				<effect trigger="happen" type="triggernews" time="2,1,1,12,18" news="21284334-3d41-4603-a629-6e153e1400fc" />
```

Comment:
triggered news not really necessary, the old definition ensured correct order as well, even related news need not be available that soon

Reply:
yes but the second news just had a tiny availability window, thus not guaranteed to show up. Because it's likely a fictional extensions without a source date I made it into a thread.